org.eclipse.m2e.core.tests/resources/projects/demo/mvnw

```diff
@@ -0,0 +1,316 @@
+#!/bin/sh
```

Comment:
maven wrapper should not be required here, please remove it to make the test as minimal as possible.